src/scripts/ci/setup_gh_actions.sh

```diff
@@ -95,8 +95,6 @@ if type -p "apt-get"; then
     elif [ "$TARGET" = "docs" ]; then
         sudo apt-get -qq install doxygen python3-docutils python3-sphinx
 
-    elif [ "$TARGET" = "format" ]; then
-        sudo apt-get -qq install clang-format-15
```

Comment:
FYI: GH Actions' Ubuntu 24.04 image brings `clang-format-17` by default. No need to explicitly install any clang-format anymore.